sale_variant_configurator/models/sale_order.py

```diff
@@ -113,11 +113,11 @@ def product_uom_change(self):
         # TODO: Handle this when there's no product_id filled
         super(SaleOrderLine, self).product_uom_change()
 
-    def _auto_init(self, cr, context=None):
+    def _auto_init(self, context=None):
         # Avoid the removal of the DB column due to sale_stock defining
```

Comment:
Is this still needed? Now everything is in new API and overwriting the field maybe is enough. Please try it (and this is for sale_stock and sale_stock_variant_configurator).